src/controllers/authentication.js

```diff
@@ -180,3 +183,63 @@ export const logout = (req, res) => {
     });
     res.status(200).json({ status: 'success', data: null });
 }
+
+export const forgotPassword = catchAsync(async (req, res, next) => {
+    const verifiedUser = await User.findOne({ where:{email: req.body.email }});
+    if (!verifiedUser) {
+      return next(new AppError('There is no user with email address.', 404));
+    }
+    const resetToken = randomBytes(32).toString('hex');
+
+    const passwordResetToken = createHash('sha256')
+      .update(resetToken)
+      .digest('hex'); 
+
+    const passwordResetExpires = Date.now() + 10 * 60 * 1000;
+
+      await verifiedUser.update({passwordResetToken,passwordResetExpires})
+      await verifiedUser.save();
+  
+    const resetURL = `${req.protocol}://${req.get(
+      'host'
+    )}/api/v1/user/resetpassword/${resetToken}`;
+  
+    try {
+      await new Email(verifiedUser, resetURL).sendPasswordReset();
+      res.status(200).json({
+        status: 'success',
+        message: 'token send to Email',
```

Comment:
can you rerun a message not too technical, average user wont know what's a token